src/ftest.jl

```diff
@@ -0,0 +1,130 @@
+type FTestTable
```

Comment:
It seems to me that we should just use `CoefTable` for this rather than defining a new type.